modules/ROOT/pages/pce-configuration.adoc

```diff
@@ -0,0 +1,17 @@
+= Configure Private Cloud Edition in Studio
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Anypoint Studio 7.4 and later support native integration with Anypoint Platform Private Cloud Edition (PCE) versions 1.6 and 2.0.
+
+To point your Studio installation to integrate with the Anypoint platform components in a PCE deployment option:
+
+. In the taskbar at the top of the Anypoint Studio display, select *Anypoint Studio* > *Preferences...*.
+. Select *Anypoint Studio* > *Anypoint Platform*.
+. Under the *PCE Settings* section, check the *Use a Private Cloud Instance URL*.
```

Comment:
```suggestion
. In the *PCE Settings* section, select *Use a Private Cloud Instance URL*.
```
Is this a checkbox? You could also word this "select the *Use a Private Cloud Instance URL* checkbox.
Checkboxes are selected and unselected.